modules/virtual_network_gateway/main.tf

```diff
@@ -1,86 +1,119 @@
+# https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/public_ip
```

Comment:
what do you think about adding this as a standard? so to all `main.tf` files?

Reply:
I agree :) I was adding this in comments to have quick link to the documentation while developing the module :)